app/models/bracket.rb

```diff
@@ -12,5 +12,36 @@ class Bracket < ApplicationRecord
   has_many :entries, dependent: :destroy
   has_many :matchups, dependent: :destroy
 
+# NEED TO MAKE AN after_create ACTION TO AUTOMATICALLY RUN THE METHODS WHEN A BRACKET IS CREATED (like in periods model of status_reporting)
+
+  # make a validation that the number of entries equals an acceptable bracket (8, for now; add 4 and 16 later; add power of 2 later), if not then the bracket needs to fail to create
+
   enum status: { created: "created", in_progress: "in_progress", completed: "completed" }
+  # enum number_of_entries: { eight: 8 } # gonna leave this in to come back to later on, might not need it/reformat it
+
+  def create_entries
+    self.number_of_entries.times do |i|
```

Comment:
If you would rather the block variable start at 1 instead of 0, you can loop over a `Range` instead

```suggestion
    (1..number_of_entries).each do |i|
```

where `i` will start at `1` and end with a value of `number_of_entries`.